extensions/functions_datetime.yaml

```diff
@@ -218,6 +218,58 @@ scalar_functions:
           - name: y
             value: interval_day
         return: timestamp
+  -
+    name: "multiply_interval"
+    description: Multiply an interval by an integral number.
+    impls:
+      - args:
+          - name: x
+            value: i8
+          - name: y
+            value: interval_day
+        return: interval_day
+      - args:
+          - name: x
+            value: i16
+          - name: y
+            value: interval_day
+        return: interval_day
+      - args:
+          - name: x
+            value: i32
```

Comment:
Postgres supports double as well.

Reply:
It is apparently the only engine around supporting this. Is it a sufficient reason to add it?

It does so by approximating a month to 30 days whenever necessary...

Reply:
Hmm, agreed, that's not an ideal behavior to have lying around.

Reply:
Yikes, thanks for investigating :)